app/assets/stylesheets/kudos/components/_header.scss

```diff
@@ -0,0 +1,128 @@
+$stripe-width: 60px;
+$font-primary: Gotham SSm A, Gotham SSm B, Helvetica Neue, Helvetica, Arial, sans-serif;
```

Comment:
I added a `$font-gotham` to variable.scss as you suggested in the other PR, if you want to use that instead